src/app/component/comments/comments.module.ts

```diff
@@ -13,6 +13,7 @@ import { LikesCounterComponent } from './components/likes-counter/likes-counter.
 import { CommentCounterComponent } from './components/comment-counter/comment-counter.component';
 import { CommentPaginationComponent } from './components/comment-pagination/comment-pagination.component';
 import { NgxPaginationModule } from 'ngx-pagination';
+import { MatProgressSpinnerModule } from '@angular/material';
```

Comment:
-- `angular` import between components.
So, organize imports:
order `angular` imports first, next other third party libs and only then imports of your application.

Reply:
Fixed